### ISSUE TITLE
sp:  roleAssignments.remove(ra.Id) example may need updating?

### ISSUE THREAD

**Issue by mikezimm** [pnp/pnpjs#1811]
### Category
- [ ] Enhancement
- [ ] Bug
- [x] Question
- [x] Documentation gap/issue

### Version

Please specify what version of the library you are using: [   "@pnp/sp": "^2.6.0",      ]

Please specify what version(s) of SharePoint you are targeting: [   Online    ]

### Expected / Desired Behavior / Question
I wanted to Break Permissions  on my library and noticed that SharePoint always adds me as the owner which I do not want.

So I followed the example in [Role Assignments Docs](https://pnp.github.io/pnpjs/sp/security/#role-assignments).

Since by default there was only one permission assignment after breaking permissions (my as Full Control), I tried to just use the sample code since "the first" as is mentioned should be that one.

So I'm not sure if it's the docs example, me or my code... but I could not get the 'remove' to work as shown in the example.

### Observed Behavior
I didn't save a screenshot of the error but below is an example the returned object (const ras).
The error it gave was that for the "roleAssignments.remove" I needed to pass both PrinicpalId and RoleDefId.  As you can see the object returned (which is equivilant to the "ras" const in docs does not contain any keys of "Id".

![image](https://user-images.githubusercontent.com/49648086/123297884-fa7c9500-d4e5-11eb-8a08-a9849aa88510.png)

### Steps to Reproduce
This is the example I was trying to use:

```typescript
// remove a role assignment
const ras = await list.roleAssignments();
// filter/find the role assignment you want to remove
// here we just grab the first
const ra = ras.find(v => true);
const r = await list.roleAssignments.remove(ra.Id);
```

### This is what I had to do to get it to work:
Basically fetch the Full Control Id and then pass it in the remove callback... the "ra.Id" in the example did not work.
```typescript
// remove a role assignment
const { Id: fullRoleDefId } = await list.roleDefinitions.getByName('Full Control').get();
const ras = await list.roleAssignments();
// filter/find the role assignment you want to remove
// here we just grab the first
const ra = ras.find(v => true);
const r = await list.roleAssignments.remove( ra.PrincipalId, fullRoleDefId  );
```

**Comment by patrick-rodgers** [comment]
Yes, docs are indeed wrong. Looks like some confusion between the roleAssignment delete and roleAssignments remove. Thanks for pointing it out, we've got it fixed for the next release. Appreciate you taking the time to help us improve!
